app/helpers/race_edition_helper.rb

```diff
@@ -0,0 +1,17 @@
+module RaceEditionHelper
+  def current_full_course_edition
+    @current_full_course_edition ||=
```

Comment:
What's the ||= operator do? What's the idea of this entire method? From the name, it seems like a getter, but looks like it returns a Presenter and not a course edition...

Reply:
`||=` is a conditional assignment. It says, if the variable already has a value, return the existing value, otherwise assign the value on the right. This is the easiest and most common way to memoize a method in Ruby, which lets us avoid performing an expensive operation (in this case, looking up the most recent full-course `race_edition`) repeatedly. https://www.honeybadger.io/blog/ruby-rails-memoization/

We're returning an instance of the RaceEdition Presenter, that's correct. That allows us to access all of the methods defined directly on the `RaceEdition` class (like `date`) and also all the display-related methods that are defined in the Presenter (like `year`).